accounts/keystore/keystore_test.go

```diff
@@ -38,7 +37,6 @@ var testSigData = make([]byte, 32)
 
 func TestKeyStore(t *testing.T) {
 	dir, ks := tmpKeyStore(t, true)
```

Comment:
@s1na Reply for https://github.com/ethereum/go-ethereum/pull/24633#discussion_r841913624. The `dir` is used here